wazo_acceptance/steps/line.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from behave import then
+from behave import when
+
+from hamcrest import assert_that
+from hamcrest import has_entries
+from hamcrest import has_item
```

Comment:
group import

Reply:
Damn, we really need a flake8 plugin for this.